README.md

```diff
@@ -54,6 +54,11 @@ If your [supported LoRa gateway](#supported-platforms) did not come with helium-
    ```
    **NOTE**: Some platform have custom package installation requirements. Refer to the developer instructions for that platform on how to install a package.
 
+   The default region of the gateway is `US915`, if your region is different you can set the right one in `/etc/helium_gateway/default.toml`. Possible values are : `US915| EU868 | EU433 | CN470 | CN779 | AU915 | AS923 | KR920 | IN865`, after updating the value you need to restart the service
```

Comment:
You should never change default.toml. Make changes only to settings.toml for local overrides

Reply:
Thanks for the hint, just updated the Readme file 👍